docker-compose.yml

```diff
@@ -46,6 +46,7 @@ services:
     volumes:
       - ./wp-content:/var/www/html/wp-content
       - ./.htaccess:/var/www/html/.htaccess
+      - ./.env:/var/www/html/.env
```

Comment:
What does this add (I'm assuming we're configuring everything through wp-config)?

Reply:
resolved this, it doesn't need to be there